src/main/java/org/jenkinsci/plugins/durabletask/FileMonitoringTask.java

```diff
@@ -443,6 +470,7 @@ public Integer invoke(File f, VirtualChannel channel) throws IOException, Interr
 
         @Override public final void stop(FilePath workspace, Launcher launcher) throws IOException, InterruptedException {
             launcher.kill(Collections.singletonMap(COOKIE, cookieFor(workspace)));
+            launcher.kill(Collections.singletonMap(COOKIE_OLD, cookieFor(workspace, true))); // To maintain backward compatibility
```

Comment:
Can calling kill in an already killed instance  have some undesirable side effects?

Reply:
AFAIK no side effects, but please would you mind elaborating what are you thinking?

Reply:
For example, let's asume this code is launched in a job that is about to be migrated but running. Will the call to kill using the new cookie raise any exception? Will it print things that may confuse the user in the log? After that if the first call to `kill` works, may the second call in an already killed job fail somehow? Would it print any stack trace or confusing information   to the log also? That kind of things

Reply:
see https://github.com/jenkinsci/durable-task-plugin/pull/141#pullrequestreview-720404766